mimic/resource.py

```diff
@@ -22,7 +22,8 @@
 )
 from mimic.rest.noit_api import NoitApi
 from mimic.rest import (fastly_api, mailgun_api, customer_api,
-                        ironic_api, glance_api, valkyrie_api)
+                        ironic_api, glance_api, valkyrie_api,
+                        elasticsearch_api)
```

Comment:
This is not a blocking comment since it's hardly *this* PR's fault, but this set of "plugin" imports is getting out of control.  We really need to address #108 soon.